recipes/gmwi2/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "gmwi2" %}
+{% set version = "1.3" %}
+
+package:
+  name: "{{ name }}"
+  version: "{{ version }}"
+
+build:
+  noarch: generic
+  number: 0
+
+source:
+  url: https://github.com/danielchang2002/GMWI2/archive/{{version}}.tar.gz
+  sha256: c7d07cde6d48044234454c7bbd0c8c97b768b0a23d38efa240e9ab29ed3ab87d
+    # path: ..
```

Comment:
can be removed

Reply:
I'm confused, why would it be alright to remove the source? I don't have the python package uploaded to pypi.

Reply:
I was referring to the comment.

Reply:
Ah ok. I've removed the comment.